noggin/tests/unit/test_middleware.py

```diff
@@ -12,3 +14,10 @@ def test_ipa_error(client, mocker):
     page = BeautifulSoup(result.data, 'html.parser')
     assert page.title
     assert page.title.string == 'IPA Error - noggin'
+
+
+def test_flask_ext(mocker):
+    init_app = mocker.patch.object(IPAErrorHandler, "init_app")
+    dummy_app = object()
+    IPAErrorHandler(dummy_app)
+    init_app.assert_called_once_with(dummy_app)
```

Comment:
This looks like it's the same as in `noggin/tests/unit/security/test_ipa_admin.py` above. Is there a reason for that, or are they slightly different?

Reply:
Ahh, it's mocking in a different object.